### PR TITLE
Modernized and updated the "Manual installation" section

### PR DESCRIPTION
**Pull request by jmvezic**
- PHP 7.2 to PHP 7.4
- Explained bash "whoami" a bit better
- PostgreSQL 10 to 11
- Explained the postgresql.conf change a bit better
- UTF8 PostgreSQL databases
- Composer 1 to Composer 2
- Drush 9 to Drush 10
- Drupal 8 to Drupal 9
- Change ownership of /var/www directory for Composer
- Clone original drupal-project git repo instead of Islandora outdated one
- run project:9.x-dev instead of 8.x-dev
- Explained Apache conf a bit better
- Tomcat 8 to Tomcat 9
- Cantaloupe 4 to Cantaloupe 5
- OpenJDK 8 to OpenJDK 11
- Install Cantaloupe as a service and standalone jar (no longer in Tomcat as a .war)
- Fedora 5 to Fedora 6
- Updated Tomcat setenv script
- Explained the Fedora errors when Syn valve is installed
- Solr 8.3 to Solr 8.9
- Search API solr 3.0 to 4.2
- Explained Solr a bit more
- Crayfish 1.1 to 2.0
- Removed Gemini (obsolete)
- Added Karaf users.properties config
- Equalize Karaf apt version with features version
- Made it easier to copy Drupal config change

## Purpose / why

The current "Manual installation" is outdated both in versions of software and in functionality (often times command do not work because some things no longer exist)

## What changes were made?

Brought Manual installation section up-to-date.

## Interested Parties

@Islandora/documentation

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ x ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ x ] Are the changes accurate, useful, free of typos, etc?
* [ x ] Does this PR update the _last updated on_ date on the documentation page?

> __Person merging__ should ensure the following
* [ x ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ x ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ x ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by seth-shaw-unlv** [comment]
First glance, this looks great, but I won't be able to test for a few weeks. If someone can test before me, that would be great, otherwise I'll try to circle back to it in a few weeks.

File: docs/installation/manual/installing_karaf_and_alpaca.md [hunk 1]
```diff
@@ -147,14 +177,15 @@ For the Karaf features we’re going to install, we need a few different reposit
...
 ```
-- `ACTIVEMQ_KARAF_VERSION`: The latest version of ActiveMQ Karaf 5.x.x; you can find this listed at the [activemq-karaf repository page](https://mvnrepository.com/artifact/org.apache.activemq/activemq-karaf) (e.g., 5.15.11 at the time of writing)
-- `APACHE_CAMEL_VERSION`: The latest version of Apache Camel 2.x.x; you can find this listed at the [apache-camel repository page](https://mvnrepository.com/artifact/org.apache.camel.karaf/apache-camel) (e.g., 2.25.0 at the time of writing)
-- `JENA_OSGI_VERSION`: The latest version of the Apache Jena OSGi features; you can find this listed at the [jena-osgi-features repository page](https://mvnrepository.com/artifact/org.apache.jena/jena-osgi-features) (e.g., 3.14.0 at the time of writing)
+- `ACTIVEMQ_KARAF_VERSION`: The version of ActiveMQ we wrote down at the beginning of this chapter when installing ActiveMQ via `apt-get`
+- `APACHE_CAMEL_VERSION`: The latest version of Apache Camel 2.x.x; you can find this listed at the [apache-camel repository page](https://mvnrepository.com/artifact/org.apache.camel.karaf/apache-camel) (e.g., 3.11.0 at the time of writing)
```
Comment (seth-shaw-unlv) [inline docs/installation/manual/installing_karaf_and_alpaca.md]: The example version (3.11.0) will not allow you to install the features. fcrepo-service-activemq requires org.apache.camel.component.jms to be "(version>=2.20.0)(!(version>=3.0.0))". So the latest acceptable version is 2.25.4.

File: docs/installation/manual/installing_karaf_and_alpaca.md [hunk 2]
```diff
@@ -147,14 +177,15 @@ For the Karaf features we’re going to install, we need a few different reposit
...
+- `ISLANDORA_KARAF_VERSION`: The latest version of Islandora Karaf 1.x; you can find this listed at the [islandora-karaf repository page](https://mvnrepository.com/artifact/ca.islandora.alpaca/islandora-karaf) (e.g., 1.0.5 at the time of writing)
+- `JENA_OSGI_VERSION`: The latest version of the Apache Jena OSGi features; you can find this listed at the [jena-osgi-features repository page](https://mvnrepository.com/artifact/org.apache.jena/jena-osgi-features) (e.g., 4.1.0 at the time of writing)
```
Comment (seth-shaw-unlv) [inline docs/installation/manual/installing_karaf_and_alpaca.md]: The example version (4.1.0) will not allow you to install the features. org.apache.jena.rdf.model needs to be (version>=3.1.0)(!(version>=4.0.0))". So the latest acceptable version is 3.17.0.

File: docs/installation/manual/installing_karaf_and_alpaca.md [hunk 3]
```diff
@@ -335,12 +365,6 @@ Before we can configure the features we’re going to use, they need to be insta
 # we're using, but is being included here to resolve the aforementioned
 # missing link in the dependency chain.
 /opt/karaf/bin/client feature:install jena
-/opt/karaf/bin/client feature:install fcrepo-camel
```
Comment (seth-shaw-unlv) [inline docs/installation/manual/installing_karaf_and_alpaca.md]: Why was this, and the rest of these removed? I needed to run them for my site to work.

**Review by seth-shaw-unlv — DISMISSED** [review]
There are a few bits with Karaf. But otherwise things check out.

File: docs/installation/manual/installing_karaf_and_alpaca.md (hunk 3 above)
Comment (jmvezic) [reply docs/installation/manual/installing_karaf_and_alpaca.md]: I was constantly getting a version mismatch error with that on Debian 10. I'll try again with a clean install. If it works for you you can revert that chunk.

Comment (seth-shaw-unlv) [reply docs/installation/manual/installing_karaf_and_alpaca.md]: Perhaps the version mismatch errors you experienced could be resolved by using the versions I mentioned in the above comments (i.e. keeping Camel @ 2.x and Jena @ 3.x)?

Comment (jmvezic) [reply docs/installation/manual/installing_karaf_and_alpaca.md]: I'll be sure to try that out soon and get back to you. That could indeed be it.

**Comment by seth-shaw-unlv** [comment]
@jmvezic, status update?

**Review by kstapelfeldt — APPROVED** [review]
I hope it is okay, but I am going ahead to merge these changes so we can get this useful work into the documentation.

**Comment by seth-shaw-unlv** [comment]
@kstapelfeldt , no, we really do need the changes that I requested for a working install. 

Although, I suppose we can merge this PR as-is if we *immediately* follow it up with a new PR to fix those feature versions.

**Comment by seth-shaw-unlv** [comment]
Turns out that I have edit permissions on the source repo, so I just made the requested changes myself. This precludes me from reviewing and merging. 

Ideally we would have someone else run through these install instructions again, but if you are comfortable with this, @kstapelfeldt, then you can squash and merge it.

**Comment by kstapelfeldt** [comment]
Sorry Seth, I think this is my lack of Github-Fu. I thought I had merged the changes you suggested. Thank you!

**Comment by seth-shaw-unlv** [comment]
No worries.

**Comment by jmvezic** [comment]
@seth-shaw-unlv Sorry for the long delay, had a lot of work since I've made this PR. Your changes are indeed needed and work on my end.

**Pull request merged**